bsb/morphologies/__init__.py

```diff
@@ -1296,6 +1436,40 @@ def _swc_to_morpho(cls, branch_cls, content, tags=None, meta=None):
     return morpho
 
 
+def _morpho_to_swc(morpho):
+    # Initialize an empty data array
+    data = np.empty((len(morpho.points), 7), dtype=object)
+    bmap = {}
+    nid = 0
+    # Iterate over the morphology branches
+    for b in morpho.branches:
+        ids = (
+            np.arange(nid, nid + len(b) - 1)
+            if len(b) > 1
+            else np.arange(nid, nid + len(b))
+        )
+        if len(b.labelsets.keys()) > 4:
```

Comment:
```suggestion
        if len(b.labelsets.keys()) > 4:  # pragma: nocover
```

Use `pragma: nocover` if you intended not to cover something in the tests

Reply:
Will do!